pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -168,9 +169,16 @@ func (c *Cluster) Start() error {
 			errorMessages = append(errorMessages, fmt.Sprintf("failed to set orchestration starting status for node %s: %+v", n.Name, err))
 			continue
 		}
-
+		devicesToUse := n.Devices
+		availDev, deviceErr := discover.GetAvailableDevices(c.context, n.Name, c.Namespace, n.Devices, n.Selection.DeviceFilter, n.Selection.GetUseAllDevices())
+		if deviceErr != nil {
+			logger.Warningf("failed to get devices for node %s cluster %s: %v", n.Name, c.Namespace, deviceErr)
```

Comment:
shouldn't we fail the install if we don't know what is available?

Reply:
there could be data dir path too. We fail only when neither device nor data dir are available.